### PR TITLE
Feature/constant files

### PR DESCRIPTION
**Pull request by hannyle**
### Description

Having a constant folder with files containing predefined values for some variables
Add **Dataset** to the summary of folder (bug https://github.com/CSCfi/metadata-submitter-frontend/issues/167)

### Related issues

https://github.com/CSCfi/metadata-submitter-frontend/issues/158
https://github.com/CSCfi/metadata-submitter-frontend/issues/167

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update (this needs a follow up PR)

### Changes Made

- Create `constants` folder and add files `folder.js`, `object.js`, `wizardStatus.js`
- Create `enums` for predefined values to use in front-end side (e.g. `ObjectTypes`, `FolderSubmissionStatus`)
- Replace hard-coded variables with `enums`

### Testing

- [x] Unit Tests
- [x] Integration Tests

### CODE REVIEW

File: src/constants/object.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+export const ObjectTypes = {
```
Comment (blankdots) [inline src/constants/object.js:1]: if we are going to add more schemas we might need to extend this. Should we be dynamically or maintain once we add more schemas ?

**Review by blankdots — COMMENTED** [review]
LGTM just want to know how we can maintain this to keep updated with some schemas and data types that might change in the backend ?

File: src/constants/object.js (hunk 1 above)
Comment (blankdots) [reply src/constants/object.js:1]: `/schemas` in the backend will give you the current schemas

**Review by blankdots — APPROVED** [review]

**Comment by hannyle** [comment]
I managed to get `schemas` from backend and converted them to `ObjectTypes` to use in front-end. It looked fine in the beginning, but it actually created more work than I had expected because I could only have them in `components` while there are cases I need the `ObjectTypes` in `unit tests` as well. 

Also there are `ObjectsArray` and a few other cases depend on `ObjectTypes` so it's better to have it as a static object like now. 

In the future if the schemas get bigger I think we can refactor it.

**Comment by blankdots** [comment]
ok ... then we can have them as is now, with the note we need to keep them in sync if we update them.

**Pull request merged**